### PR TITLE
new trip button

### PR DESCRIPTION
**Pull request by Whishes**

### CODE REVIEW

**Review by fabbrob — APPROVED** [review]
no u

File: database/schema.sql [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
+
+CREATE TABLE holidays (
+  id SERIAL PRIMARY KEY,
+  user_id INT,
+  holiday_name TEXT,
+  date_start DATE,
+  date_end DATE,
+  location_name TEXT,
+  img_link TEXT
+)
```
Comment (elisabethmj) [inline database/schema.sql]: semi colon missing

File: database/schema.sql [hunk 2]
```diff
@@ -1,6 +1,36 @@
...
-);
\ No newline at end of file
+);
+
+CREATE TABLE holidays (
+  id SERIAL PRIMARY KEY,
+  user_id INT,
+  holiday_name TEXT,
+  date_start DATE,
+  date_end DATE,
```
Comment (Whishes) [inline database/schema.sql:12]: dsfsfsf

**Review by elisabethmj — COMMENTED** [review]
semi colons missing

File: database/schema.sql (hunk 1 above)
Comment (Whishes) [reply database/schema.sql]: added

**Pull request closed without merging**